### PR TITLE
Add transterm HP package

### PR DESCRIPTION
**Pull request by erasche**

### CODE REVIEW

File: packages/package_transtermhp_2_09/tool_dependencies.xml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                    <destination>$INSTALL_DIR/bin</destination>
+                </action>
+                <action type="move_file">
+                    <source>expterm.dat</source>
+                    <destination>$INSTALL_DIR/data</destination>
+                </action>
+                <action type="set_environment">
+                  <environment_variable action="prepend_to" name="PATH">$INSTALL_DIR/bin/</environment_variable>
+                  <environment_variable action="prepend_to" name="TRANSTERM_EXPTERM_DATA">$INSTALL_DIR/data/expterm.dat</environment_variable>
+                  <environment_variable action="set_to" name="TRANSTERM_PATH">$INSTALL_DIR/</environment_variable>
```
Comment (bgruening) [inline packages/package_transtermhp_2_09/tool_dependencies.xml]: Eric, can you add `TRANSTERMHP_ROOT_PATH`

File: packages/package_transtermhp_2_09/tool_dependencies.xml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                    <source>mfold_rna.sh</source>
+                    <destination>$INSTALL_DIR/bin</destination>
+                </action>
+                <action type="move_file">
+                    <source>expterm.dat</source>
+                    <destination>$INSTALL_DIR/data</destination>
+                </action>
+                <action type="set_environment">
+                  <environment_variable action="prepend_to" name="PATH">$INSTALL_DIR/bin/</environment_variable>
+                  <environment_variable action="prepend_to" name="TRANSTERM_EXPTERM_DATA">$INSTALL_DIR/data/expterm.dat</environment_variable>
```
Comment (bgruening) [inline packages/package_transtermhp_2_09/tool_dependencies.xml]: This needs to be `set_to`

File: packages/package_transtermhp_2_09/tool_dependencies.xml (hunk 1 above)
Comment (erasche) [inline packages/package_transtermhp_2_09/tool_dependencies.xml]: should I use that in lieu of `TRANSTERM_PATH`?

File: packages/package_transtermhp_2_09/tool_dependencies.xml (hunk 2 above)
Comment (erasche) [inline packages/package_transtermhp_2_09/tool_dependencies.xml]: yikes, thanks

File: packages/package_transtermhp_2_09/tool_dependencies.xml (hunk 1 above)
Comment (bgruening) [inline packages/package_transtermhp_2_09/tool_dependencies.xml]: If `TRANSTERM_PATH` is not needed by the program, yes please.

**Pull request merged**